server/src/main/java/org/apache/gravitino/server/web/rest/ModelOperations.java

```diff
@@ -0,0 +1,410 @@
+  public Response getModel(
+      @PathParam("metalake") String metalake,
+      @PathParam("catalog") String catalog,
+      @PathParam("schema") String schema,
+      @PathParam("model") String model) {
+    LOG.info("Received get model request: {}.{}.{}.{}", metalake, catalog, schema, model);
+    NameIdentifier modelId = NameIdentifierUtil.ofModel(metalake, catalog, schema, model);
+
+    try {
+      return Utils.doAs(
+          httpRequest,
+          () -> {
+            Model m = modelDispatcher.getModel(modelId);
+            LOG.info("Model got: {}", modelId);
+            return Utils.ok(new ModelResponse(DTOConverters.toDTO(m)));
+          });
+
+    } catch (Exception e) {
+      return ExceptionHandlers.handleModelException(OperationType.GET, model, schema, e);
+    }
+  }
+
+  @POST
+  @Produces("application/vnd.gravitino.v1+json")
+  @Timed(name = "register-model." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
+  @ResponseMetered(name = "register-model", absolute = true)
+  public Response registerModel(
+      @PathParam("metalake") String metalake,
+      @PathParam("catalog") String catalog,
+      @PathParam("schema") String schema,
+      ModelRegisterRequest request) {
+    LOG.info(
+        "Received register model request: {}.{}.{}.{}",
+        metalake,
+        catalog,
+        schema,
+        request.getName());
+
+    try {
+      request.validate();
+      NameIdentifier modelId =
+          NameIdentifierUtil.ofModel(metalake, catalog, schema, request.getName());
+
+      return Utils.doAs(
+          httpRequest,
+          () -> {
+            Model m =
+                modelDispatcher.registerModel(
+                    modelId, request.getComment(), request.getProperties());
+            LOG.info("Model registered: {}", modelId);
+            return Utils.ok(new ModelResponse(DTOConverters.toDTO(m)));
+          });
+
+    } catch (Exception e) {
+      return ExceptionHandlers.handleModelException(
+          OperationType.REGISTER, request.getName(), schema, e);
+    }
+  }
+
+  @DELETE
+  @Path("{model}")
+  @Produces("application/vnd.gravitino.v1+json")
+  @Timed(name = "delete-model." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
+  @ResponseMetered(name = "delete-model", absolute = true)
+  public Response deleteModel(
+      @PathParam("metalake") String metalake,
+      @PathParam("catalog") String catalog,
+      @PathParam("schema") String schema,
+      @PathParam("model") String model) {
+    LOG.info("Received delete model request: {}.{}.{}.{}", metalake, catalog, schema, model);
+    NameIdentifier modelId = NameIdentifierUtil.ofModel(metalake, catalog, schema, model);
+
+    try {
+      return Utils.doAs(
+          httpRequest,
+          () -> {
+            boolean deleted = modelDispatcher.deleteModel(modelId);
+            if (!deleted) {
+              LOG.warn("Failed to delete model {} under schema {}", model, schema);
```

Comment:
return false means the model does not exist, not failed to delete

Reply:
I see, I will fix it.